### ISSUE TITLE
training details of all_dla34.pth provided

### ISSUE THREAD

**Issue by WHAOOOOO** [ifzhang/FairMOT#77]
The paper mentioned that the batch_size is 12, and the learning rate is 1e-4. Why the batch_size is 8 in the project by default, and the learning rate is also 1e-4? And did the all_dla34.pth model provided do fine-tuning on MOT15?

**Comment by ifzhang** [comment]
I train the model with batchsize 8 and 12 and find it can get similar results. I do not train MOT15 on all_da34.pth. So you need to finetune it on MOT15 if you want to test it on MOT15.
